src/content/docs/ra-primer/ra-in-action.mdx

```diff
@@ -9,9 +9,9 @@ In the previous section, we introduced the mathematical notation for some of the
 In this section, we will go through a few slightly more advanced uses of relational algebra to get a feel for how it works. Rather than introducing more mathematical notation, we will start using Bmg for our examples.
 
 :::note
-Although we use Bmg in this section and the two remaining ones, we are only interested in presenting the fundamentals of relational algebra. The code shown here does not represent best practices for using Bmg!
+Although we use Bmg in this section and the two remaining ones, we are only interested in presenting the fundamentals of relational algebra in its basic form.
```

Comment:
This line does not prevent shipping. The point is that the code shown is adapted to a tutorial style, rather than best practices.

Reply:
It does not prevent shipping. You can even ignore my comment.

Why presenting code that, on very intent, does not represent best practices. A mystery to me.